day-finder/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+
+
+  <script>
+
+    function dayFinder(stringData){
+
+      const date = new Date(stringData)
+      const getDay = date.getDay()
+      const day = ['sunday', 'monday', 'tuesday', 'wednessday', 'thursday', 'friday', 'saturday']
+      for (var i=0; i<day.length; i++){
```

Comment:
This loop is unnecessary. You can just have your return statement directly, you are simply accessing the array at the index your `getDay` variable is providing.